tests/hazmat/primitives/test_pkcs12.py

```diff
@@ -282,6 +292,42 @@ def _load_ca(backend):
     reason="PKCS12 unsupported in FIPS mode. So much bad crypto in it."
 )
 class TestPKCS12Creation:
+    @pytest.mark.parametrize(
+        ("ktype", "kparam"),
+        [
+            pytest.param(
+                ed448.Ed448PrivateKey,
+                None,
+                marks=pytest.mark.supported(
+                    only_if=lambda backend: backend.ed448_supported(),
+                    skip_message="Requires OpenSSL with Ed448 support",
+                ),
+            ),
+            pytest.param(
+                ed25519.Ed25519PrivateKey,
+                None,
+                marks=pytest.mark.supported(
+                    only_if=lambda backend: backend.ed25519_supported(),
+                    skip_message="Requires OpenSSL with Ed25519 support",
+                ),
+            ),
+            (rsa.RSAPrivateKey, (65537, 1024)),
+            # (rsa.RSAPrivateKey, (65537, 2048)),
```

Comment:
We can drop all the commented out RSA/DSA options -- the core test here is whether it can serialize the key types.